### DISCUSSION TITLE
Build only seems to work on Windows

### DISCUSSION THREAD

**Discussion by jgsuess** [useocl/use#48] [General]
Hi,

I have been trying to build `use` for experimentation with some XMI adapters. The build failed as below. Do you have a container or CI build, or a build environment spec (less preferred)?

Best,

JGS

```sh
[INFO] 
[ERROR] Tests run: 119, Failures: 119, Errors: 0, Skipped: 0
[INFO] 
[INFO] 
[INFO] --- maven-failsafe-plugin:2.22.2:verify (default) @ use-gui ---
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for use 7.0.1:
[INFO] 
[INFO] use ................................................ SUCCESS [  0.163 s]
[INFO] use-core ........................................... SUCCESS [ 10.492 s]
[INFO] use-gui ............................................ FAILURE [01:32 min]
[INFO] use-assembly ....................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:43 min
[INFO] Finished at: 2022-10-26T11:28:02-04:00
[INFO] ------------------------------------------------------------------
[log.txt](https://github.com/useocl/use/files/9871065/log.txt)
------

```

**Comment by jgsuess** [comment]
[log.txt](https://github.com/useocl/use/files/9871123/log.txt)

**Comment by jgsuess** [comment]
Seems the build is not cross-platform. I have JDK 15 at hand. Could either give me permissions to create an actions spec or create one that works on Ubuntu? This is challenging to reproduce...

![image](https://user-images.githubusercontent.com/782737/198092630-e122daad-7312-47c7-847e-c8457c33a4bc.png)

**Reply by h-man2** [reply]
Fixed with #49.

**Comment by h-man2** [comment]
Problem is located at integration tests: the used diff-library reports the difference in line breaks between Windows and Linux.
I will look this up.
